vllm/distributed/device_communicators/cuda_communicator.py

```diff
@@ -99,7 +99,7 @@ def all_reduce(self, input_):
             torch.distributed.all_reduce(out, group=self.device_group)
         return out
 
-    def reduce_scatter(self, input_: torch.Tensor, dim: int = -1):
+    def reduce_scatter(self, input_: torch.Tensor, dim: int = -1, sizes: Optional[List[int]] = None):
```

Comment:
Could you add a `reduce_scatterv` function instead of modifying `reduce_scatter`? I'd like to keep things symmetric between reduce-scatter/all-gather

Reply:
Thanks for reviewing, done.